frontend/src/pages/connectionTypes/manage/ConnectionTypeDataFieldModal.tsx

```diff
@@ -107,23 +107,24 @@ export const ConnectionTypeDataFieldModal: React.FC<Props> = ({
 
   return (
     <Modal
-      isOpen
+      isOpen={isOpen}
```

Comment:
Oh we actually had a recent conversation about this. All modals must supply `isOpen` at all times and we use conditional rendering. 
Please remove the incoming prop `isOpen` and return this to the previous change.